server/notification-providers/pushplus.js

```diff
@@ -0,0 +1,54 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { DOWN, UP } = require("../../src/util");
+
+class PushPlus extends NotificationProvider {
+    name = "PushPlus";
+
+    /**
+     * @inheritdoc
+     * @param {BeanModel} notification Notification object
+     * @param {string} msg Message content
+     * @param {?object} monitorJSON Monitor details
+     * @param {?object} heartbeatJSON Heartbeat details
+     * @returns {Promise<string>} Success message
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+        const url = "https://www.pushplus.plus/send";
```

Comment:
Funny off-topic note: their docs always use http for this ^^

(don't change this part. https is good)